commons/commons-test/src/main/java/com/clicktravel/common/test/BeanAssert.java

```diff
@@ -132,61 +133,125 @@ private static Object getPropertyValue(final Class<?> propertyClass) {
             }
         }
 
+        if (propertyClass.equals(List.class)) {
```

Comment:
```List.class``` should be supported in a similar the same way as ```Set.class``` is on line 109, unless there is a compelling reason otherwise.